roles/load_aware_undeploy_trimaran/tasks/main.yml

```diff
@@ -1,3 +1,25 @@
+- name: Create the src artifacts directory
+  file:
+    path: "{{ artifact_extra_logs_dir }}/src/"
+    state: directory
+    mode: '0755'
+
+- name: Define the Trimaran setup config
+  set_fact:
+    trimaran_setup_config: "{{ artifact_extra_logs_dir }}/src/trimaran-templated-config.yaml"
+
+- name: Define the same default settings from deploy role
+  include_vars: "{{ trimaran_deploy_defaults }}"
+
+- name: Dummy Thanos Endpoint
+  command: echo "NA"
+  register: thanos_endpoint
```

Comment:
this construct is required because you have this in the template:
```
address: https://{{ thanos_endpoint.stdout }}
```

I prefer doing it this way (in the deploy code), but your solution works as well:
```
- name: Get Thanos Endpoint
  shell:
    oc get route thanos-querier -n openshift-monitoring -o json
      | jq -r '.spec.host'
register: thanos_endpoint_cmd

- name: set thanos endpoint variable
  set_fact:
    thanos_endpoint: " https://{{ thanos_endpoint_cmd.stdout }}"
```
and then in the template, keep it simple:
```
address: {{ thanos_endpoint }}
```